dev/com.ibm.ws.jdbc/src/com/ibm/ws/rsadapter/jdbc/WSJdbcProxyMethod.java

```diff
@@ -129,6 +129,8 @@
         // oracle.jdbc.pool.OracleConnectionPoolDataSource.createPooledConnectionBuilder
         // oracle.jdbc.xa.client.OracleXADataSource.createPooledConnectionBuilder
         // oracle.jdbc.xa.OracleXADataSource.createPooledConnectionBuilder
+        unsafeMethods.add("createShardingKeyBuilder");
```

Comment:
Why is this blocked?  I don't see why there would be a concern with the user using an Oracle interface to construct a sharding key.

Reply:
You are correct, it shouldn't be...